realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -112,7 +113,6 @@ public final AllTypesColumnInfo clone() {
         proxyState.setConstructionFinished();
     }
 
-    @Override
```

Comment:
"iff" is a mathematical term (set theory, I'd guess?) and I thought it was more widely used.  Fixed